install/kubernetes/helm/istio/charts/pilot/values.yaml

```diff
@@ -14,6 +14,8 @@ sidecar: true
 traceSampling: 1.0
 # if protocol sniffing is enabled. Default to false.
 enableProtocolSniffing: false
+# timeout for protocol detection. Pilot will use default filter chain when times out
```

Comment:
default is..? TCP?

Reply:
Yes. The default filter chain is TCP proxy without any filter chain match.